.github/workflows/pkg-platform.yml

```diff
@@ -125,6 +125,7 @@ jobs:
         compression: [xz, gz]
     runs-on: ubuntu-latest
     env:
+      AWS: ${{ inputs.dry-run && 'aws' || 'echo '}}
```

Comment:
ooh, very slick.

Reply:
if it had worked. hm. i wonder if it's needs the `== true`. might explain why the other `input.dry-run &&` things look like they're not working.

Reply:
oh. ha. just logic reversed. i'll fix it.